trin-core/src/portalnet/protocol.rs

```diff
@@ -97,6 +123,32 @@ impl JsonRpcHandler {
                         .collect();
                     let _ = cmd.resp.send(Ok(Value::Array(routing_table_info)));
                 }
+                DummyHistoryNetworkData => {
+                    let (resp_tx, mut resp_rx) = mpsc::unbounded_channel::<Result<Value, String>>();
+                    let message = HistoryNetworkEndpoint {
+                        kind: HistoryEndpointKind::GetHistoryNetworkData,
+                        resp: resp_tx,
+                    };
+                    self.history_tx.as_ref().unwrap().send(message).unwrap();
+                    let response = match resp_rx.recv().await.unwrap() {
```

Comment:
I'm not really clear. When might this panic in any of these 3 `unwrap()`s? If it's remotely possible, then we need some handling.

(and the same below, of course)

Reply:
Yup, for sure there's a lot of error handling that I just kinda glossed over so far. I'll go back and implement some handling